### PR TITLE
Better splitting of the mighty edpm-deploy.yml playbook

### PR DESCRIPTION
**Pull request by cjeanner**
Splitting it like that would allow to get a cleaner overview of the file
content, allowing an easier maintenance later.

This patch also introduces some more top-level parameters, allowing to
activate or not RHOL/CRC and Libvirt supports.

### CODE REVIEW

**Comment by cjeanner** [comment]
recheck

Weird.... "merge conflict" while branch is 100% up-to-date compared to main

**Comment by cjeanner** [comment]
/hold

Need to check something first - I think I can make it even better.

File: .zuul.d/collect-logs.yml [hunk 1]
```diff
@@ -15,13 +15,29 @@
...
+      ansible.builtin.stat:
+        path: "{{ ansible_user_dir }}/ci-framework"
+
+    - name: Copy ci-framework interesting files
+      when:
+        - cifmw_state.stat.exists | bool
+      ansible.builtin.shell:
+        chdir: "{{ ansible_user_dir }}/zuul-output/logs/"
+        cmd: |
+          mkdir ci-framework; pushd ci-framework
```
Comment (raukadah) [inline .zuul.d/collect-logs.yml:29]: Can we convert directory creation to ansible and then call shell module to copy log files? We can use block for that if we use ansible.

**Review by raukadah — APPROVED** [review]
Looks good, few nit picks to address.

File: .zuul.d/collect-logs.yml (hunk 1 above)
Comment (cjeanner) [reply .zuul.d/collect-logs.yml:29]: right - #88 will fix this.

**Pull request merged**